crates/uv-python/fetch-download-metadata.py

```diff
@@ -264,6 +266,11 @@ async def _fetch_downloads(self, pages: int = 100) -> list[PythonDownload]:
                         and download.triple.libc == "musl"
                     ):
                         continue
+                    if (
+                        download.release == CPYTHON_BAD_LINUX_RUNTIME_START
```

Comment:
Doesn't this need to be `>=`?

Reply:
(you also need to run the script)

Reply:
Ah haha, good catch. I widened the range and reran which without `>=` effectively reverted the changes to the json 😓